CHANGELOG.rst

```diff
@@ -17,6 +17,12 @@ History
 - Unify tracer fixture and move it to conftest.py
 - Fix some deprecation warnings
 - Clean up imports
+- Add ability to set ``response_handler_hook`` using ``set_response_handler_hook`` method of ``RequestsPatcher``
+- Add base Patcher
+- Refactor opentracing_instrumentation/client_hooks/requests.py
+- Add test for the requests
+- Simplify the singleton decorator
```

Comment:
L21-24: don't need to include implementation details in release notes (I realized above are bad examples too).

Reply:
I've removed those lines from the changelog.